resources/85-tessel.rules

```diff
@@ -0,0 +1 @@
++ATTRS{idVendor}=="1209", ATTRS{idProduct}=="7551", ENV{ID_MM_DEVICE_IGNORE}="1", MODE="0666"
```

Comment:
Oops - that's what happens when you're jet lagged and writing code while being approached with questions from every angle. Fixed in a subsequent commit. Thanks!